config/initializers/new_framework_defaults.rb

```diff
@@ -1,20 +0,0 @@
-# Be sure to restart your server when you modify this file.
```

Comment:
This file should be retained

This actually changes behaviours - the first two have no effect in an API app but the third may do. They are old defaults and should probably be dropped but in a separate PR since they do not relate to the switch to Zeitwerk